### ISSUE TITLE
2.11.3 patch release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3733]
- #3708 
- #3692
- #3695 
- #3720 
- #3721 
- #3703
- #3729 
- #3732
- #3727 
- #3728
- #3719 

## Pick list
```
git cherry-pick db79e2e53edefdbe97ff1a48db049405e46f45e5
git cherry-pick 2592b7ef112435522298f0c866802ff658e1ebf4
git cherry-pick c83d4f77a3b4b96dd645e992cd8af35b401ae957
git cherry-pick 54927d38dadf595b7e5da5533da57432c4bb0ae2
git cherry-pick cdff8e54e7954f3eff7e19e900c1d883e45ec183
git cherry-pick 1ab9b1b08650403a4d20a5ce5df589d4902e8083
git cherry-pick 01b31cf54ad69dbc2a9e2d49f60623d5d07667fb
git cherry-pick 3259a3c95e5762845deb9028b284956cc25a65f0
git cherry-pick fcc7f8dd4a2f4806678fbdb81012af3ccd923e04
git cherry-pick a69a47f612cd3fdd0fd9a13a92857939f4ed0797
git cherry-pick ba082a03de9e3fa910e47e1c1c7b7be3d23aa380
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.11.2..HEAD --pretty='format:- %s'`
```
- Closes #3718 Improve styling of AZ Finder exposed form, especially for multiple vocabulary filters (#3719)
- Closes #3725: Hard code higher WebP conversion quality setting until configurable for GD in core. (#3728)
- For #3705 Menu blocks intermittently not visible to anonymous users after updating to Quickstart 2.11 (#3727)
- Close #3730 mark minified JS as minified. (#3732)
- Closes #3723 Card title on image is red when the card is not clickable (#3729)
- Update drupal/viewsreference requirement from 2.0.0-beta9 to 2.0.0-beta10 (#3703)
- Update drupal/date_ap_style requirement from 2.0.1 to 2.0.2 (#3721)
- Update drupal/optional_end_date requirement from 1.3.0 to 1.4.0 (#3720)
- Update citation-style-language/locales requirement from 0.0.20 to 0.0.22 (#3695)
- Update drupal/smart_date requirement from 4.1.5 to 4.1.6 (#3692)
- Update citation-style-language/styles requirement from 0.0.447 to 0.0.474 (#3708)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.11.3
